### PR TITLE
PWX-33692 Add new CLI to trigger checksum validation

### PR DESCRIPTION
**Pull request by nikhil-bhupale**
libopenstorage changes to add new CLI
pxctl volume verify-checksum start
pxctl volume verify-checksum stop
pxctl volume verify-checksum status

Verified above command with all possible options

### CODE REVIEW

File: api/api.proto [hunk 1]
```diff
@@ -6102,3 +6104,114 @@ message SdkVolumeCatalogResponse {
...
+  // OpenStorageVerifyChecksum service APIs()
+  enum VerifyChecksumStatus {
+    // VerifyChecksum operation is an unknown state
+    VERIFY_CHECKSUM_UNKNOWN = 0;
+    // VerifyChecksum operation is not running for the specified volume
+    VERIFY_CHECKSUM_NOT_RUNNING = 1;
+    // VerifyChecksum operation started for the specified volume
+    VERIFY_CHECKSUM_STARTED = 2;
+    // VerifyChecksum operation was stopped by the user for the specified volume
+    VERIFY_CHECKSUM_STOPPED = 4;
```
Comment (talakad) [inline api/api.proto:6163]: Shouldn't this be set to 3?

File: api/server/sdk/verify_checksum.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+) (*api.SdkVerifyChecksumStartResponse, error) {
+
+	if s.driver(ctx) == nil {
+		return nil, status.Error(codes.Unavailable, "Resource has not been initialized")
+	}
+	var err error
+	if len(req.GetVolumeId()) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Must supply a volume id")
+	}
+
```
Comment (talakad) [inline api/server/sdk/verify_checksum.go:50]: may be try checkAccessFromDriverForVolumeIds ?

File: api/api.proto (hunk 1 above)
Comment (nikhil-bhupale) [reply api/api.proto]: Fixed

File: api/server/sdk/verify_checksum.go (hunk 2 above)
Comment (nikhil-bhupale) [reply api/server/sdk/verify_checksum.go:50]: Fixed

**Review by talakad — APPROVED** [review]

**Pull request merged**